### PR TITLE
[Wallet] Make reveal error during verification recoverable

### LINKED ISSUE #2337: “Pending Invitation” section is not shown on the “Wallet” tab even when a user has invited other users.
**Frequency:** 100%
**App version:** IOS test flight build v1.5.2 (18)
**Repro on:** iPhone 7 (13.3), iPhone 7+ (12.4), iPad pro 2nd Gen (13.2.2)
**Testing Account** 914 745 2564/ [Backupkey](https://docs.google.com/spreadsheets/d/1Iy41GGA6e2pES2XHjBn7nb8X3n8zZsu7EW1jvwD2e70/edit?usp=sharing)

**Repro Steps:** 
1) Launch the app. 
2) Tap on the “Cog” icon. 
3) Tap on the “Invite” option. 
4) Send an invitation to any non-celo user. 
5) Make sure an invitation is sent and the Invitation sent activity is shown with an Invitation fee. 
6) Observed.
 
**Note:**I have wait up to 1 day but still did not get the Pending Invitation section. 

**Current Behavior:** “Pending Invitation” section is not shown 
**Expected Behavior:** “Pending Invitation” section should be shown 
**Attachment:** IOS_pendingInvitation.png

IOS_pendingInvitation

### LINKED ISSUE #2593: User is unable to return to verification flow to paste codes
### Expected Behavior

add a ‘dismiss’ option to the "verification failed" modal so that someone who already received sms codes can input them

### Current Behavior
Re-request of codes fails without giving the user the option to enter their codes if they have received them

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

This case is hit when you successfully requested attestations but interrupted the process before inputting the codes. If you try again after an hour, you'll request new attestations. But if you try again *before* an hour, reveal will fail. so we offer a way for you to enter the codes anyway, assuming you got them the first time. 

Reduce Escrow payment reclaim time to 1 day based on feedback from a few people

### Tested

Did verification successfully
Forced an error early in verification
Skipped code input stage of verification and then used Enter Codes option during second run

### Related issues

- Fixes #2593 
- Fixes #2337 

### Backwards compatibility

Yes

ver-error-enter-codes

### CODE REVIEW

File: packages/mobile/src/verify/VerificationFailedModal.tsx [hunk 1]
```diff
@@ -3,34 +3,54 @@ import colors from '@celo/react-components/styles/colors'
...
+  verificationStatus: VerificationStatus
+  cancelVerification: typeof cancelVerification
 }
 
-type Props = OwnProps & WithTranslation
+export function VerificationFailedModal(props: Props) {
+  const { t } = useTranslation(Namespaces.nuxVerification2)
+  const [isDismissed, setIsDismissed] = React.useState(false)
+
+  const onDismiss = () => {
```
Comment (i1skn) [inline packages/mobile/src/verify/VerificationFailedModal.tsx]: As far as I understand, that would create new lambda every time something changes in props, right? Shall we use `useCallback` here?

File: packages/mobile/src/verify/VerificationFailedModal.tsx [hunk 2]
```diff
@@ -3,34 +3,54 @@ import colors from '@celo/react-components/styles/colors'
...
+    setIsDismissed(true)
+  }
 
-function VerificationFailedModal(props: Props) {
   const onSkip = () => {
```
Comment (i1skn) [inline packages/mobile/src/verify/VerificationFailedModal.tsx]: same as above

**Review by i1skn — COMMENTED** [review]
Overall looks great! See just small comments I left.

File: packages/mobile/src/verify/VerificationFailedModal.tsx (hunk 2 above)
Comment (jmrossy) [reply packages/mobile/src/verify/VerificationFailedModal.tsx]: Done :)

File: packages/mobile/src/verify/VerificationFailedModal.tsx (hunk 1 above)
Comment (jmrossy) [reply packages/mobile/src/verify/VerificationFailedModal.tsx]: Done :)

**Review by i1skn — APPROVED** [review]
🚢 it!

**Pull request merged**